### PR TITLE
Updated some names for clarity

### PR DESCRIPTION
**Pull request by ADepic**
if diet:xyz = yes
The name was changed from xyz to Some xyz options
eg
diet:vegetarian=yes
name was "vegetarian"
now is "some vegetarian options"
Same done for takeaway and delivery

Also Bicycle Parking name improvements

Stands ---> Sheffield Stands
Wall Loops ---> Wheel Benders

These names are more common among cyclists.

### CODE REVIEW

File: OsmAnd/res/values/phrases.xml [hunk 1]
```diff
@@ -1820,9 +1820,9 @@
 
 	<string name="poi_public_bath">Public bath</string>
 
-	<string name="poi_male_yes">Male</string>
+	<string name="poi_male_yes">Male Only</string>
```
Comment (vshcherb) [inline OsmAnd/res/values/phrases.xml]: I don't think this is correct cause there are toiltets with male/yes + female/yes

**Comment by ADepic** [comment]
Directly from the OSM wiki:
Key:female

A value other than no implies no for all other gender-based access tags, but this can be overridden by adding more access role tags. In case of conflict, resolve in favour of more access.

Key:male

A value other than no is interpreted as yes, and implies no for all other unspecified gender-based access tags, but this can be overridden by adding more access role tags. In case of conflict, resolve in favour of more access.

**Comment by vshcherb** [comment]
Please revert those changes cause it will take quite a lot of time and it is a complex techinical task to provide translations for a tag combinations

File: OsmAnd/res/values/phrases.xml (hunk 1 above)
Comment (jaller94) [reply OsmAnd/res/values/phrases.xml]: The OSM wiki actually uses the strict word "only" as well. But as mentioned, multiple gender-keys can be `yes` on one POI.
https://wiki.openstreetmap.org/wiki/Key:male

For an app like OSMAnd, I suggest: Male allowed / Female allowed

**Pull request closed without merging**